frappe/integrations/doctype/ldap_settings/ldap_settings.py

```diff
@@ -85,7 +85,7 @@ def sync_roles(self, user, additional_groups=None):
 		needed_roles = set()
 		needed_roles.add(self.default_role)
 
-		lower_groups = [g.lower() for g in additional_groups]
+		lower_groups = [g.lower() for g in additional_groups or []]
```

Comment:
Instead of this
```
lower_groups = [g.lower() for g in additional_groups or []]
```

You can do like
```
def sync_roles(self, user, additional_groups=[]):
```

Reply:
I've tried that, codacy check didn't allow empty array as default value for funtion parameter. That was the first commit.